FAQ: What is a presentation? (#107)
tlsnotary/docs-mdbook: src/faq.md

No, the TLSNotary protocol does not solve the "Oracle Problem." The Oracle Problem refers to the challenge of ensuring that off-chain data used in blockchain smart contracts is trustworthy and tamper-proof. While TLSNotary allows a Prover to cryptographically authenticate TLS data to a designated Verifier, trust is still required in the designated Verifier when it attests to the verified data on-chain. Therefore, this is not a trustless, decentralized solution to the Oracle Problem.

TLSNotary can be used to bring data on-chain, but when the stakes are high, it is recommended to combine TLSNotary with a dedicated oracle protocol to mitigate these risks. Multiple projects are currently exploring the best solutions.

### What is a presentation in TLSNotary? { #faq13 }

> In TLSNotary, a **presentation** is the step where the Prover selectively reveals specific parts of the data they committed to during the attestation phase. By using these earlier commitments, the Prover can choose to disclose only particular segments of the data while keeping other parts hidden or redacted. This enables a flexible and controlled way to share proofs, ensuring that sensitive information remains private.

Reviewer: I assume you were trying to avoid confusing the term "data" since it can mean both the content of the presentation and the content on the TLS transcript. I'm suggesting to use "TLS data".

```suggestion
In TLSNotary, a **presentation** means data shared by the Prover to selectively reveal specific parts of the TLS data committed to earlier during the attestation phase. By using these earlier commitments, the Prover can choose to disclose only particular segments of the TLS data while keeping other parts hidden or redacted. This enables a flexible and controlled way to share proofs, ensuring that sensitive information remains private.
```
